### PR TITLE
RHDEVDOCS-6249: Add support for step actions in cluster resolver

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `pipelines-docs-1.17`

Issue: [RHDEVDOCS-6249](https://issues.redhat.com/browse/RHDEVDOCS-6249)

Link to docs preview: https://84484--ocpdocs-pr.netlify.app/openshift-pipelines/latest/create/remote-pipelines-tasks-resolvers.html

QE review:
- [x] QE has approved this change.

**Additional information:**

### CODE REVIEW

File: modules/op-resolver-cluster.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ When creating a pipeline run, you can specify a pipeline that exists on the same
   - name: namespace
     value: <namespace>
   - name: kind
-    value: [pipeline|task]
+    value: [pipeline|task|step]
```
Comment (ppitonak) [inline modules/op-resolver-cluster.adoc:25]: @vdemeester shouldn't it be stepaction?

Comment (vdemeester) [reply modules/op-resolver-cluster.adoc:25]: yes, `stepaction` (see [here](https://github.com/tektoncd/pipeline/blob/main/docs/cluster-resolver.md#parameters))

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Comment by mramendi** [comment]
I had a read, looks great!

**Review by ppitonak — APPROVED** [review]

**Review by xenolinux — COMMENTED** [review]
LGTM

**Review by xenolinux — COMMENTED** [review]
merge review LGTM

**Pull request merged**